### ISSUE TITLE
Bug report: Application crashed with KeyError

### ISSUE THREAD

**Issue by csadorf** [aiidalab/aiidalab-qe#259]
## Automated report

_This issue was created with the app's automated bug reporting feature.
Attached to this issue is the full traceback as well as an environment
fingerprint that contains information about the operating system as well as all
installed libraries._

## Additional comments (optional):

I submitted a pure-relaxation calculation for Silica.

## Attachments

<details>
<summary>Traceback</summary>

```python-traceback
~/apps/quantum-espresso/aiidalab_qe/widgets.py in _observe_node(self, change)
    376                 clear_output()
    377                 if change["new"]:
--> 378                     display(viewer(change["new"]))
    379 
    380 

~/.local/lib/python3.8/site-packages/aiidalab_widgets_base/viewers.py in viewer(obj, downloadable, **kwargs)
     77             )
     78             return obj
---> 79         raise exc
     80 
     81 

~/.local/lib/python3.8/site-packages/aiidalab_widgets_base/viewers.py in viewer(obj, downloadable, **kwargs)
     69     try:
     70         _viewer = AIIDA_VIEWER_MAPPING[obj.node_type]
---> 71         return _viewer(obj, downloadable=downloadable, **kwargs)
     72     except (KeyError) as exc:
     73         if obj.node_type in str(exc):

~/apps/quantum-espresso/aiidalab_qe/node_view.py in __init__(self, node, **kwargs)
    427             """
    428         )
--> 429         self.workflows_summary = SummaryView(self.node)
    430 
    431         self.summary_tab = ipw.VBox(children=[self.workflows_summary])

~/apps/quantum-espresso/aiidalab_qe/node_view.py in __init__(self, wc_node, **kwargs)
    193             return "Yes" if truthy else "No"
    194 
--> 195         report = generate_report_dict(self.wc_node)
    196 
    197         env = Environment()

~/apps/quantum-espresso/aiidalab_qe/report.py in generate_report_dict(qeapp_wc)
    132 def generate_report_dict(qeapp_wc):
    133     """Generate a dictionary for reporting the inputs for the `QeAppWorkChain`"""
--> 134     return dict(_generate_report_dict(qeapp_wc))
    135 
    136 

~/apps/quantum-espresso/aiidalab_qe/report.py in _generate_report_dict(qeapp_wc)
     31     # Properties
     32     run_relax = builder_parameters["relax_type"] != "none"
---> 33     run_bands = builder_parameters["run_bands"]
     34     run_pdos = builder_parameters.get("run_pdos", False)
     35 

KeyError: 'run_bands'
```
</details>

<details>
<summary>Environment fingerprint</summary>
<pre>eNpVWF1v47oR_SuB-3plWJLjfLxdpO1Dsdtum90WRVFcUBJtM6ZIhqTiKBfb3975oq0gTz6kyJkzM2eG-X31pmMy3q0e619Wwaq8
93FcPf6-UrE_mqz7PEW9evzParftTF79svrTlz-v_gtb53z07rfL16t2fb-uW9hwhf5QN_X_6vv1Zruuqx_d5PJ08_z1283343Tz
VcWbZntT3z029WPb3vz4_nTTbJpm9RPOVv1JHXQiM4wZVNV7NGJVr3frB7iCwZD0NHiANwC3F_h1UnDRqFOIOiVPpt2um7JuVQdQ
06zv1vUCq7R1dFT9aWv1qsvuzWe4Ovt46o_KODLs0-rZDAedU9WpxGa36y2t-yqYkwJoB2zxPUD3K9nYsj12eNHjTKY8CKLHzvR0
zj0jLpnmmEcrGN0dwmBiImTLt2EA33QVVD7ScQ1_HA_eNVW_3xv0DG3bfIarzrjBuEPi9UbWoz_T6Q0bTp6BzQ9MfZpdXx2001Fl
H2ljTd_lHOWgLR3UQXB7Za2YRFAf55DpOD4cDFBx9tH5TNvYoc5q1aMrt-
sNudL5k8bfDWxAI7rXYOWLulm3BEWfrQnMJ8fwCQ7RT97l6G3ZW-NxPS5k7y3auwVTtgQWB2AXfd_rmA2TB_kKqVffIsp8gte3vO
uo4qCzWFuQpHPlIOLKmg8dyXQ59KhH4_b-k6W9if2U2EbO1t6a_gTAnaQp_YbqGIPVmYtuI8leltzeHKq9sVoKZXNZS8E4R1Zg1l
O0em99VKMS1skx7wZFhNQcLQIqqdHqqNxgIVkW6cl3-u5FnN8R5k9G91POmpPjjiJGgfeHqMIRHW_Bb2Z97m2xjAl6Ir3hpIGUQw
8H1YNAiSzgnkF304EYRIS2aFAOSchbdJKw_ZT08D5aoZpBUAuINXNYovLHF-UOngD4u8d9LoViSSMVNUz2bHqpMbAfWdNQtNplqb
0GYoLu6lEZW71B9IdLlfAZsBeo8MZljjeSoLNuucbIF_3e64AGHqKfAovOehN7NGuvUlbBcIj5LoRekncJU0uxUzu6C5PBesojVB
08ew_lcM38luttP7H2YxBYrShQb-ajuFoTprWzlOm1WGoGp_hw_DEGH6EGu2rUWYHfn5Lpugpi7afYo-pDqBr-Nsz4gcgp2wZ3BJ
9yzcsnHZ22JKhQeAJKfO7Qkc0VQnmasrFpIT24BA4s9BGQ6313kh5XsNLvWbtBDxI8PORFD6bQhLv_YtyLapjymhx5IaTKZtSLy1
98B54LcQRAvCgJRBtawaLe82fy-xLTtpw0gR86VlDYkElMB1C8XOEOukWL6iu-6FZiBy6ezPmqmbcCJG_fpHg5P6yKJ9ABELVYdH
RDcIZU9VB65ANnCebbRYRqVvmv6uQXJQATwWnwZ54lthdoCs9qrxdHjSqjzDNzpbVfwco40CO5iBgcTcqTIPdk-QjpPhcBB6XfER
Z1BXoSSx3gOEMOjWHkMi6WjumA8iclSAdONpvB9Ew9B2qc3CnqQivuch1II3DIXYGL0XU4cCmGOF2dTrmihmq80N0SnHHkeCcq7i
ks7mDfjD4TDxve5LPuQGvJDhZwN42iiQ2PGN7qSyi26LoHWRnVIJztOHEKeM2pwhbayPLPut8wzQEGlZxVZ7UIC-8bXwPnsvyG5p
FkJroVwPdgDzDP-K1shD5kTr5UkWDJSxj5xnws0SqDW-hY6x_4C_0eNEVlK3NSOKRJSpC2QyO0OkFr1ot0_2aspWnnQerrm3H5wg
59Fngk4PBT6s3Le62QTjJJiJ-9dGf6GT3I4VFP6covnM5pj4shV-
jayeQSXfosoYDJRbQ1zb0Ph6bikUlyA4Ma8gzngJ6mxUQR5g6Kk-3aCtIrE6UOmeZv85PZ_-
NfRBqnHmzySeXFqA3IpfKBSTIFtNplGVTZum_zAbtg4iA2ciFkOyhxSahGdn6dn__-RcbelpC_qie7zIj5b0G75-cvhXkG0QzOQ2
TpgTBAUr6QyqIc5jRC1iepWr70GWQpyTzCm6hRQNPRQnQjg7qsJDsdzH4WhS0L0EFe0jLL5-xfejUGSUzBouKUaMuHH0RCXRz-96
9fv4gOsDEf9DRoisrDo6Oke8O9H8_B_pvjLDXVEkJNVRWB3N0T-DqBsPDHDXNSsIpGXyH_drkwetE5tjHu-_bh_k7uomMnNWq7np
WMUzCoUjYs8HXPWo01t7ssVbhUlelyg-37Hp1MvYGkr2DWj64kasO4yBiXWNL6FIouP6zvAHqGxtx8jyodFw8jGLmnUGjbFSrTUV
scW49qFIllFHibJjNIkmA2JYPDNXZdnsXuyIdk3lnYefJM4XDp5DtiEJL5V4ttugSm2X5Cqx-XSaTlL9KrpZKS2Zt8ziqiqOqiDg
xGaDQVhqJtH7i4EZ6LzNec3SDFE_ZiEUya76EocGieqb3JNh3h5aHkIS2vlwzBv7wbdwK952pyBkfqQV61CEPyqyEAv322Mq3Sdl
8SgrshjLtyCU_m-XUYSWF2TD0Uh8mWx5BbKSPCQMZ0Wsxn8Bvi9huNYfh_hjIW0gcfMGcoLlvcOzl8rMETPRs6A427I9ygEzhb4p
gGxLKaTGEPFVZJsEsxTNFeH22Lhgw4BL1l6dzxG_KfCkpvXvSjs8r9cfCHwgVmwbmHkevyKMcrzrqjp5dMCw1D2oGV8hDHxoEXnI
9aM7EtSxZPb1UHfTRdnr8N1aYsBR2Nx-
Hk0pvZW5n7XHfhkha5w51h1JeZnh2B99Jl7KzLXDtDfi5efh8mhPKvoNXPn_8Hx0cnlw==</pre>
</details>

**By submitting this issue I confirm that I am aware that this information can
potentially be used to determine what kind of calculation was performed at the
time of error.**

**Comment by csadorf** [comment]
This bug occurred with a fresh installation of the app version v22.07.0 .

**Comment by unkcpz** [comment]
fixed by #260
